### DISCUSSION TITLE
给了root权限，结果app还是说root授权失败

### DISCUSSION THREAD

**Discussion by wushendutang** [gkd-kit/gkd#771] [General]
![log-1731342209790.zip](https://github.com/user-attachments/files/17705613/log-1731342209790.zip)

![Screenshot_2024-11-12-00-16-55-268_li.songe.gkd.jpg](https://github.com/user-attachments/assets/46b2269a-6d11-40c7-a61a-96934066ad9f)

![Screenshot_2024-11-12-00-17-25-216_tnfi.z.pbvy.ewx.jpg](https://github.com/user-attachments/assets/60bbdccf-cc76-41f9-9d86-cd836d3cbfcf)

（原来图片这个栏能上传文件啊）

**Comment by lisonge** [comment]
我没有已 ROOT 的设备，代码是参考其它开源代码得到的

<https://github.com/WuDi-ZhanShen/AccessibilityManager/blob/8c5bc6f6c0a1e3e09ff580a6bf7bc0d1a89082b5/app/src/main/java/com/accessibilitymanager/MainActivity.java#L580>

你可以使用 shizuku 授权

---

此外欢迎 pr 修复此问题

**Comment by China54188** [comment]
去找模块版本爱玩机工具箱里面好像有
